rust/arrow/src/compute/kernels/take.rs

```diff
@@ -1109,6 +1187,62 @@ mod tests {
         test_take_list_with_nulls!(i64, LargeList, LargeListArray);
     }
 
+    #[test]
+    fn test_take_fixed_size_list() {
+        do_take_fixed_size_list_test::<Int32Type>(
+            3,
+            vec![
+                Some(vec![Some(0), Some(1), Some(2)]),
+                Some(vec![Some(3), Some(4), Some(5)]),
+                Some(vec![Some(6), Some(7), Some(8)]),
+            ],
+            vec![2, 1, 0],
```

Comment:
would it be possible / useful to add a test that uses some null indices?

Reply:
Really appreciate your review!

The third case used null indices (`2` in `[3, 2, 1, 2, 0]`) if you mean to null value inside a list.

BTW I also changed the first case to include some nulls in sub-array in the following commit.